src/main/kotlin/eu/kevin/api/models/general/getProjectSettings/ForceRedirectToBankMode.kt

```diff
@@ -0,0 +1,11 @@
+package eu.kevin.api.models.general.getProjectSettings
```

Comment:
I think our models package naming is bit weird. Package is named as verb instead of noun. `getPaymentStatus` should be just `paymentStatus`, `getProjectSettings` should be just `projectSettings`.
Let's change that, because current naming is quite unusual.